package.json

```diff
@@ -13,46 +13,48 @@
     },
     "homepage": "https://github.com/fluid-project/handbook.floeproject.org#readme",
     "scripts": {
-        "build": "npm-run-all -l clean build:*",
-        "build:webpack": "mix --production",
+        "build": "npm-run-all -l clean build:eleventy",
         "build:eleventy": "cross-env NODE_ENV=production eleventy",
-        "clean": "rimraf dist",
+        "clean": "rimraf _site",
         "dev": "cross-env NODE_ENV=development eleventy --serve",
-        "lint": "fluid-lint-all",
-        "start": "npm-run-all -l clean -p start:*",
-        "start:webpack": "mix watch",
-        "start:eleventy": "npm run dev",
+        "lint": "run-s -l lint:*",
+        "lint:css": "stylelint \"**/*.css\"",
+        "lint:js": "eslint \"**/*.js\"",
+        "lint:markdown": "markdownlint-cli2 \"**/*.md\"",
+        "lint:yml": "eslint --no-error-on-unmatched-pattern \"**/*.yml\"",
+        "start": "npm-run-all -l clean -p dev",
         "prepare": "husky install"
     },
     "dependencies": {
-        "@11ty/eleventy": "1.0.1",
-        "@11ty/eleventy-navigation": "0.3.3",
-        "@11ty/eleventy-plugin-syntaxhighlight": "4.0.0",
-        "eleventy-plugin-fluid": "1.0.0",
-        "infusion": "4.1.0"
+        "@11ty/eleventy": "2.0.1",
+        "@11ty/eleventy-navigation": "0.3.5",
+        "@11ty/eleventy-plugin-syntaxhighlight": "5.0.0",
+        "eleventy-plugin-fluid": "2.3.1",
+        "infusion": "4.6.0"
     },
     "devDependencies": {
-        "@babel/core": "7.17.10",
-        "@commitlint/cli": "17.0.2",
-        "@commitlint/config-conventional": "17.0.3",
+        "@commitlint/cli": "17.6.7",
+        "@commitlint/config-conventional": "17.6.7",
         "cross-env": "7.0.3",
-        "fluid-lint-all": "1.2.2",
+        "eslint": "8.46.0",
+        "eslint-config-fluid": "2.1.1",
+        "eslint-plugin-jsdoc": "46.4.5",
+        "eslint-plugin-yml": "1.8.0",
         "github-slugger": "1.4.0",
-        "html-minifier": "4.0.0",
-        "husky": "8.0.1",
-        "image-size": "1.0.1",
-        "laravel-mix": "6.0.44",
-        "linkedom": "0.14.9",
-        "markdown-it-anchor": "8.6.4",
-        "move-file": "2.1.0",
+        "husky": "8.0.3",
+        "linkedom": "0.14.26",
+        "lint-staged": "13.2.3",
+        "markdownlint-cli2": "0.8.1",
+        "markdownlint-config-fluid": "0.1.5",
         "npm-run-all": "4.1.5",
-        "postcss": "8.4.16",
-        "postcss-dir-pseudo-class": "6.0.4",
-        "postcss-logical": "5.0.4",
-        "resolve-url-loader": "5.0.0",
         "rimraf": "3.0.2",
-        "sass": "1.54.3",
-        "sass-loader": "13.0.2",
-        "slugify": "1.6.5"
+        "stylelint": "14.16.1",
+        "stylelint-config-fluid": "1.0.0",
+        "stylelint-use-logical-spec": "5.0.0"
```

Comment:
I think you'll need `stylelint-config-standard-scss`.

Reply:
I am noticing a lot of deprecation errors when running the `lint:scss` command. I think you used a newer version of `stylelint-config-standard-scss` which is not compatible with `stylelint-config-fluid`. You will need to use the version [here](https://github.com/inclusive-design/idrc/blob/0e0c634eff4b493d3b1d7a81cba1e6c03f0a6376/package.json#L82), and also add a dependency lock for `stylelint-config-standard-scss` in the Dependabot configuration.